src/components/MinMaxZoomField.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useEffect, useRef } from "react";
+import durationStyles from "../forms/DurationField.module.css";
+import formStyles from "../styles/Forms.module.css";
+import inputStyles from "../styles/Input.module.css";
+import {isValidIntegerZeroOrLarger} from '../form/validators';
+
+export interface MinMax {
+  minZoom: number,
+  maxZoom: number
+};
+
+interface Props {
+  name: string,
+  value: MinMax,
+  valueChanged: Function,
+  triedToSubmit: boolean,
+}
+
+const isValidMaxZoom = (min:any, max: any) => {
+    const minZoomValidated = isValidIntegerZeroOrLarger(min);
+    const maxZoomValidated = isValidIntegerZeroOrLarger(max);
```

Comment:
For the whole min max fields, I made an integer input fields. I think we can reuse them to make these fields without making this whole component for the min and max values. In this case, the validation would also be easier. The MinMaxZoomField.tsx component should also stay in the `form` director instead of the `component` director as it is used for the form specifically.

Reply:
"For the whole min max fields, I made an integer input fields" > I merged your branch, but backed up the old minmax component (as commented out code). 
I think the min max how you made it is better then my component so for now I agree.

Anyway I am not sure if I believe that the validation and management of different subfields of a larger combination-field (in this case minmax) should be completely managed  by the form (as opposed to the field itself).

If the form manages it all then we have to keep repeating some form validation code in every form we use that combination field in.

I also see that the form fields are now set up very generic so setting up 2 seperate fields is more "compatible" with the form. So yes I do clearly see advantages for now.